application.js

```diff
@@ -16,6 +16,15 @@ app.use(cors());
 app.use('/sys', mbaasExpress.sys(securableEndpoints));
 app.use('/mbaas', mbaasExpress.mbaas);
 
+/* uncomment this code if you want to use $fh.auth in the app preview
+ * specify function for localAuth
```

Comment:
perhaps explain this a little bit more clear. The localAuth is only used for local development. If the app is deployed on the platform, this function will be ignored and the request will be forwarded to the platform to perform authentication